data-migration/migrate-sql-file-tidb.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Migrate Data from SQL Files to TiDB
+summary: Learn how to migrate data from SQL Files to TiDB.
+---
+
+# Migrate Data from SQL Files to TiDB
+
+This document describes how to migrate data from MySQL SQL files to TiDB using TiDB Lightning. For how to generate MySQL SQL files, refer to [Export to SQL Files](https://docs.pingcap.com/tidb/stable/dumpling-overview#export-to-sql-files).
+
+## Prerequisites
+
+- [Install TiDB Lightning using TiUP](/migration-tools.md)
+- [Get the target database privileges required for TiDB Lightning](/tidb-lightning/tidb-lightning-faq.md#what-are-the-privilege-requirements-for-the-target-database)
+
+## Step 1. Prepare SQL files
+
+Put all the SQL files in the same directory, like `/data/my_datasource/`. TiDB Lighting recursively searches for all `.sql` files in this directory and its subdirectories.
+
+## Step 2. Define the target table schema
+
+Because SQL files do not contain schema information, before importing data into TiDB, you need to create the target table schema. You can create the target table schema by either of the following two methods:
+
+* **Method 1**: create the target table schema using TiDB Lightning.
+
+    1. Write SQL files that contain the required DDL statements.
+
+        - The format of file name is `${db_name}-schema-create.sql`, and this file should have `CREATE DATABASE` statements.
+        - The format of file name is `${db_name}.${table_name}-schema.sql`, and this file should have `CREATE TABLE` statements.
+
+    2. During the migration, add the following configuration in `tidb-lightning.toml`:
```

Comment:
I suggest remove this sub step. For tidb-lightning < v5.2.0, no-schema = false is the default value, so user needn't explicitly set it. For >= v5.2, this config is deprecated and needn't set anymore.